docs/Makefile

```diff
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    = -j auto
+SPHINXBUILD   = sphinx-build
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+    @$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
running this makefile gave me an error:
`Makefile:19: *** missing separator.  Stop.`

Could be that your using spaces instead of tabs
https://stackoverflow.com/questions/920413/make-error-missing-separator

Reply:
I actually just copied this from PyBaMM haha, and you get the same error there, but the docs seem to build ok?

Reply:
yea, docs build ok if you manually call sphinx-build (which is the command in the readme), but the Makefile doesn't work. You could either just delete the Makefile or fix it, up to you.

Reply:
probably same over in pybamm if you copied it from there :)